homeassistant/components/volvooncall/__init__.py

```diff
@@ -49,68 +40,6 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-CONFIG_SCHEMA = vol.Schema(
-    vol.All(
-        cv.deprecated(DOMAIN),
-        {
-            DOMAIN: vol.Schema(
-                {
-                    vol.Required(CONF_USERNAME): cv.string,
-                    vol.Required(CONF_PASSWORD): cv.string,
-                    vol.Optional(
-                        CONF_SCAN_INTERVAL, default=DEFAULT_UPDATE_INTERVAL
-                    ): vol.All(cv.time_period, vol.Clamp(min=DEFAULT_UPDATE_INTERVAL)),
-                    vol.Optional(CONF_NAME, default={}): cv.schema_with_slug_keys(
-                        cv.string
-                    ),
-                    vol.Optional(CONF_RESOURCES): vol.All(
-                        cv.ensure_list, [vol.In(RESOURCES)]
-                    ),
-                    vol.Optional(CONF_REGION): cv.string,
-                    vol.Optional(CONF_SERVICE_URL): cv.string,
-                    vol.Optional(CONF_MUTABLE, default=True): cv.boolean,
-                    vol.Optional(CONF_SCANDINAVIAN_MILES, default=False): cv.boolean,
-                }
-            )
-        },
-    ),
-    extra=vol.ALLOW_EXTRA,
-)
-
-
-async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
-    """Migrate from YAML to ConfigEntry."""
-    if DOMAIN not in config:
-        return True
-
-    hass.data[DOMAIN] = {}
-
-    if not hass.config_entries.async_entries(DOMAIN):
-        new_conf = {}
-        new_conf[CONF_USERNAME] = config[DOMAIN][CONF_USERNAME]
-        new_conf[CONF_PASSWORD] = config[DOMAIN][CONF_PASSWORD]
-        new_conf[CONF_REGION] = config[DOMAIN].get(CONF_REGION)
-        new_conf[CONF_SCANDINAVIAN_MILES] = config[DOMAIN][CONF_SCANDINAVIAN_MILES]
-        new_conf[CONF_MUTABLE] = config[DOMAIN][CONF_MUTABLE]
-
-        hass.async_create_task(
-            hass.config_entries.flow.async_init(
-                DOMAIN, context={"source": SOURCE_IMPORT}, data=new_conf
-            )
-        )
-
-        async_create_issue(
-            hass,
-            DOMAIN,
-            "deprecated_yaml",
-            breaks_in_ha_version=None,
-            is_fixable=False,
-            severity=IssueSeverity.WARNING,
-            translation_key="deprecated_yaml",
```

Comment:
These strings haven't been cleaned up in this PR. Please create a PR to do so.

Reply:
They have been removed in this PR

Reply:
WTH I've looked twice... Sorry about the noise 🙏